deploy/kicbase/Dockerfile

```diff
@@ -32,7 +32,7 @@ FROM ubuntu:focal-20210401
 ARG BUILDKIT_VERSION="v0.9.0"
 ARG FUSE_OVERLAYFS_VERSION="v1.7.1"
 ARG CONTAINERD_FUSE_OVERLAYFS_VERSION="1.0.3"
-ARG CRIO_VERSION="1.22"
+ARG CRIO_VERSION="1.22.1"
```

Comment:
This actually stays as `1.22`, when they release a point release they replace the `1.22.0` binary with `1.22.1`

Reply:
Sounds good and thank you for the details.